### PR TITLE
project_forecast_line: improve documentation

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: project_forecast_line/readme/CONFIGURATION.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+In the settings, you'll find a Forecase Management section where you can configure
```
Comment (yankinmax) [inline project_forecast_line/readme/CONFIGURATION.rst]:
```suggestion
In the settings, you'll find a Forecast Management section where you can configure
```

File: project_forecast_line/readme/CONFIGURATION.rst [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+* the granularity of the forecast (day / week / month)
+* the horizon of the forecast (number of months)
+* if we want to manage forecast from quotations
+
+Then you can configure in the Project application the forecast roles. These are
```
Comment (yankinmax) [inline project_forecast_line/readme/CONFIGURATION.rst]:
```suggestion
Then you can configure in the Forecast application the forecast roles. These are
```

File: project_forecast_line/README.rst [hunk 3]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
+
+  * for costs (project tasks for instance) we take the absolute value of the
+    forecast hours (so it is a positive number)
+
+  * for resources (employee capacity for a period), we take the capacity and
+    substract all the costs for that employee on the same period. So it will be
+    positive if the employee still has some free time, and negative if he is
+    overloaded with work.
+
+
```
Comment (yankinmax) [inline project_forecast_line/README.rst:75]:
```suggestion

** for the note: consolidated forecast is computed in days using forecast hours values accordingly to ease report readability. 
```

File: project_forecast_line/README.rst [hunk 4]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
 .. contents::
    :local:
 
+Usage
+=====
+
+Forecast lines have the following data:
+
+* Forecast hours: it is positive for resources (employees) and negative for
+  things which consume time
```
Comment (yankinmax) [inline project_forecast_line/README.rst:60]:
```suggestion
  things which consume time (project tasks for instance)
```

File: project_forecast_line/README.rst [hunk 5]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
 
 This module allows to plan your resources using forecast lines.
-For each employee of the company, the module will generate forecast line records with a positive capacity based on their working time schedules. Then, tasks assigned to employees will generate forecast lines with a negative capacity which will "consume" the work time capacity of the employees.
-Forecast lines also come in two states "forecast" or "confirmed", depending on whether the consumption is confirmed or not. For instance, holidays requests and sales quotation lines create lines of type "forecast", whereas tasks for project which are in a running state create lines with type "confirmed".
+
+For each employee of the company, the module will generate forecast line
+records with a positive capacity based on their working time schedules. Then,
+tasks assigned to employees will generate forecast lines with a negative
+capacity which will "consume" the work time capacity of the employees.
+
```
Comment (yankinmax) [inline project_forecast_line/README.rst:34]:
```suggestion

It is basically done by the cron scheduled to run each day. So, to start using consolidated forecast report you first need to set everything mentioned in Usage section. Then, probably run Forecast recomputation cron manually from Scheduled Actions or wait till cron creates records.

```

File: project_forecast_line/README.rst [hunk 6]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
+The idea is that you can then see the work capacity and scheduled work of
+people by summing the "forecasts" per time period. If you have more resources
```
Comment (yankinmax) [inline project_forecast_line/README.rst:36]:
```suggestion
people by summing the "forecasts" per time in the period. If you have more resources
```

File: project_forecast_line/README.rst [hunk 7]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
+sum. Otherwise you are in trouble and need to recruit or reschedule your
+work. Another way to use the report is checking when the work capacity of a
+department becomes positive (or high enough) in order to provide you potential
+customers with an estimate of when a project would be able to start.
+
+Forecast lines also come in two states "forecast" or "confirmed", depending on
+whether the consumption is confirmed or not. For instance, holidays requests
+and sales quotation lines create lines of type "forecast", whereas tasks for
+project which are in a running state create lines with type "confirmed".
+
```
Comment (yankinmax) [inline project_forecast_line/README.rst:47]:
```
To get the best experience using the Forecast application you may want to install:
 * project_forecast_line_holidays_public module which takes public holidays into account during forecast lines creation
 * project_forecast_line_bokeh_chart module which improves the reports of project_forecast_line module by using the bokeh widget available in OCA/web

File: project_forecast_line/README.rst [hunk 8]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
+  linked to the employee.
+
+* draft sale orders (if enabled in the settings) will create forecast lines of
+  type "forecast" for each sale order line having a product with a forecast
+  role and start and end dates. The forecast hours are negative
+
+* confirmed sale orders don't create forecast lines. This is handled by the
+  tasks created at the confirmation of the sale order
+
+* project tasks create forecast lines if they have a linked role and start/end
```
Comment (yankinmax) [inline project_forecast_line/README.rst:90]:
```suggestion
* project tasks create forecast lines if they have a linked role and planned start/end
```

File: project_forecast_line/README.rst [hunk 9]
```diff
@@ -26,14 +26,86 @@ Project Forecast Lines
...
+  date. The type of the line will depend on the related project's stage. The
+  forecast quantity is based on the remaining time of the task, which is spread
```
Comment (yankinmax) [inline project_forecast_line/README.rst:92]:
```suggestion
  forecast hours quantity is based on the remaining time of the task, which is spread
```

**Review by yankinmax — APPROVED** [review]
Thanks @gurneyalex for the great description done. I've left some minor suggestions.

**Review by gjotten — APPROVED** [review]
Really nice writeup @gurneyalex!

**Pull request merged**